### ISSUE TITLE
"unexpected output during activation" notice on plugin activation

### ISSUE THREAD

**Issue by sumobi** [pippinsplugins/Restrict-Content-Pro#106]
When I activate the plugin locally (2.0.8) I get this:

    The plugin generated 2009 characters of unexpected output during activation. If you notice “headers already sent” messages, problems with syndication feeds or other issues, try deactivating or removing this plugin.

Seems to be caused by these lines in `rcp_options_upgrade()`. Commenting these out prevents the notice from appearing.

	/****************************************
	* upgrade discount codes DB
	****************************************/

	if( !$wpdb->query( "SELECT `max_uses` FROM `" . $rcp_discounts_db_name . "`" ) ) {
		$wpdb->query( "ALTER TABLE `" . $rcp_discounts_db_name . "` ADD `max_uses` mediumint" );
		update_option( 'rcp_discounts_db_version', $rcp_discounts_db_version );
	}
	if( ! $wpdb->query( "SELECT `expiration` FROM `" . $rcp_discounts_db_name . "`" ) ) {
		$wpdb->query( "ALTER TABLE `" . $rcp_discounts_db_name . "` ADD `expiration` mediumtext" );
		update_option( 'rcp_discounts_db_version', $rcp_discounts_db_version );
	}
	if( ! $wpdb->query( "SELECT `subscription_id` FROM `" . $rcp_discounts_db_name . "`" ) ) {
		$wpdb->query( "ALTER TABLE `" . $rcp_discounts_db_name . "` ADD `subscription_id` mediumint" );
		update_option( 'rcp_discounts_db_version', $rcp_discounts_db_version );
	}

**Comment by cliffordp** [comment]
Uploaded Master, renamed folder to `restrict-content-pro`, added to site, activated plugin: `The plugin generated 2247 characters of unexpected output during activation. If you notice “headers already sent” messages, problems with syndication feeds or other issues, try deactivating or removing this plugin.`

**Comment by pippinsplugins** [comment]
Hmm okay, I'll dig into it further.

**Comment by cliffordp** [comment]
Turns out I didn't rename it. I meant to ;)
I had the name of `restrict-content-pro v2.1.4` as the folder name.
Everything worked fine except I noticed https://pippinsplugins.com/wp-content/uploads/2015/02/Screen-Shot-2015-02-11-at-1.33.52-PM.png showed user levels draggable to re-order and mine didn't. Opened console, lots of errors, renamed directory, reactivated plugin, and still got the "unexpected characters" message but now there were no more console errors and subscription levels were draggable.
